src/main/java/blusunrize/immersiveengineering/common/blocks/metal/TileEntityBelljar.java

```diff
@@ -217,8 +222,10 @@ else if(renderActive)
 							int out = Math.min(inventory.get(j).getCount(), 16);
 							ItemStack stack = Utils.copyStackWithAmount(inventory.get(j), out);
 							stack = Utils.insertStackIntoInventory(outputTile, stack, facing);
-							if (!stack.isEmpty())
+							if (!stack.isEmpty())  {
```

Comment:
1. Wrong codestyle, the { should be on the next line.
2. Does this work if the output is removed through the GUI? You don't seem to be resetting `outputFull` in that case.

Reply:
ah need to fix that